roles/jd-server/src/lib/job_declarator/mod.rs

```diff
@@ -100,6 +125,16 @@ impl JobDeclaratorDownstream {
                                 Self::send(self_mutex.clone(), message).await.unwrap();
                             }
                             Ok(SendTo::None(_)) => (),
```

Comment:
I know that this was already like that but we are trying to make this code more stable and prod ready, so catch all should disappear if you see one from the prototype phase just remove it is few line of code

Reply:
In this case the catchall is dangerous is not like doing Error::ErrorKind(_) where you want to log it and fail for all the error of errorkind. In this case SendTo::None(m) is used for different thing so it is better to explicitly handle them, maybe we should rethink the SendTo type not sure, for now I would leave it like that and explicitly handle all the path

Reply:
Yes but here in particular is very vague what to do. For example, I can track all the types of SendTo returned by the handler, handle them and report an error if arrives a different SendTo.

Reply:
done